.evergreen/download-mongodb.sh

```diff
@@ -525,9 +515,34 @@ get_mongodb_download_url_for ()
    fi
 
    if [ "$VERSION_INCLUDES_CRYPT_SHARED" = "YES" ]; then
+      # Use the same version as the server.
+      REQUESTED_CRYPT_SHARED_VERSION="$_VERSION"
```

Comment:
maybe it makes sense to print what version we actually choose?

Reply:
It is printed in [run-orchestration.sh](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/3b12064cc469fc18d8024d48353d29b89dd26472/.evergreen/run-orchestration.sh#L105-L117).

Also, I am unsure if there are callers of this script that rely on `get_mongodb_download_url_for` only printing the server URL. E.g. a caller may do this:
```sh
# Get the server download URL.
MONGODB_URL=$(get_mongodb_download_url_for $DISTRO $VERSION)
```